moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -511,6 +511,30 @@ void RobotState::setJointGroupPositions(const JointModelGroup* group, const Eige
   updateMimicJoints(group);
 }
 
+void RobotState::setJointGroupActivePositions(const JointModelGroup* group, const std::vector<double>& gstate)
+{
+  assert(gstate.size() == group->getActiveVariableCount());
+  std::size_t i = 0;
```

Comment:
Random tidbit: you don't need `std::`

`size_t` alone is fine. There are already some places in moveit_core which do this. For example:

robot_state/test/robot_state_benchmark.cpp:    `for (size_t i = 0; i < runs; ++i)`

But of course there is no harm leaving it as is. [Stackoverflow link](https://stackoverflow.com/a/5813736/3499467)